docs/anomaly-detection/GUI.md

```diff
@@ -0,0 +1,326 @@
+---
+weight: 2
+title: GUI
+menu:
+  docs:
+    parent: "anomaly-detection"
+    identifier: "vmanomaly-gui"
+    weight: 2
+    title: GUI
+tags:
+  - metrics
+  - enterprise
+  - guide
+aliases:
+- /anomaly-detection/gui.html
+- /anomaly-detection/gui
+- /anomaly-detection/GUI
+- /anomaly-detection/GUI/
+---
+
+## Introduction
+
+{{% available_from "v1.26.0" anomaly %}} `vmanomaly` is shipped with a built-in [vmui-like](https://play.victoriametrics.com/) [GUI](https://en.wikipedia.org/wiki/Graphical_user_interface) that provides an intuitive interface for rapid exploration of how anomaly detection models, their configurations and included domain knowledge impacts the results of anomaly detection, before such configurations are deployed in production.
```

Comment:
I'd suggest to use a link to the documentation instead of playground here - https://docs.victoriametrics.com/victoriametrics/single-server-victoriametrics/#vmui